packages/forms/src/model.ts

```diff
@@ -1455,7 +1455,7 @@ export class FormGroup extends AbstractControl {
   }
 
   /**
-   * Add a control to this group.
+   * Add a control to this group. Nothing is done if control already exists with the given name unlike `setControl`.
    *
    * This method also updates the value and validity of the control.
```

Comment:
I'd propose updating docs a bit:
```suggestion
   * Add a control to this group.
   *
   * If a control with a given name already exists, it would *not* be replaced with a new one. If you want
   * to replace an existing control, use the `setControl` method instead.
   * This method also updates the value and validity of the control.
```